modules/cli/pages/cbepctl/set-flush_param.adoc

```diff
@@ -112,10 +112,13 @@ The following are the command options:
 | `couch_response_timeout`
 | timeout in receiving a response from CouchDB.
 
+| `exp_pager_enabled`
+| Checks if the expiry pager task is enabled. The value is set to true by default.
```

Comment:
Is "Checks" the right word here (not sure if this has come from kv internal docs)? I think this should be "Sets" in terms of how it works...

Reply:
Thanks. It should be "Sets"